modules/core/src/geometry/geometry.js

```diff
@@ -0,0 +1,119 @@
+import {uid, assert} from '../utils';
+
+// Rendering primitives - specify how to extract primitives from vertices.
+// NOTE: These are numerically identical to the corresponding WebGL/OpenGL constants
+export const DRAW_MODE = {
```

Comment:
Should this not just be pulled in from `@luma.gl/constants`?

Reply:
Again the idea is that geometry is a math class without knowledge of webgl